coralproject/talk
# Remove Dashboard for the interim

## What does this PR do?
- Removes all Dashboard components and references, until we can relaunch it with some much needed love <3

## How do I test this PR?

- Open the Admin
- Make sure you can't see the Dashboard
- Default page should still be Moderate
- Make sure no other errors
